elementary/config/config.py

```diff
@@ -91,3 +94,7 @@ def _validate_elementary_profile(self):
                 raise NoElementaryProfileError
         except FileNotFoundError:
             raise NoProfilesFileError(self.profiles_dir)
+
+    def validate_timezone(self):
```

Comment:
Could you please make this a private method? Only the commands should be exposed (`validate_monitor()`, `validate_report()`, etc.)

```suggestion
    def _validate_timezone(self):
```

Reply:
done.